classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def new_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
+end
```

Comment:
[ OPTIONAL CHANGE ]

- [ ] For this class, you're expected to have an attribute reader for a student. Kindly add that. This was not highlighted by the previous reviewer but I highly recommend you add it.

`attr_reader :students`